challenges/objects-arrays.js

```diff
@@ -6,12 +6,47 @@
   object name, diet, weight, length, period
 */
 
+function Dino(attr) {
```

Comment:
Nice constructor function, could also have them just be objects but I like this better.